src/rules/at-conditional-rule-no-parentheses/__tests__/index.js

```diff
@@ -0,0 +1,53 @@
+import rule, { ruleName, messages } from "..";
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  syntax: "scss",
+  fix: true,
+
+  accept: [
+    {
+      code: "@if true {}",
+      description: "accepts @if with no parentheses"
+    },
+    {
+      code: "@while true {}",
+      description: "accepts @while with no parentheses"
+    },
+    {
+      code: `@if true {}
+      @else if true {}`,
+      description: "accepts @else if with no parentheses"
+    },
+    {
+      code: `@if true {}
+      @else if true {}
+      @else {}`,
+      description: "accepts @else unconditionally"
+    }
```

Comment:
Maybe test that `@if (1 + 1) == 2` is allowed?

Reply:
Great idea! Added + made a fix.